docs/provisioning_guide/src/asciidoc/_chapters/requirements.adoc

```diff
@@ -415,11 +456,34 @@ java.heapsize^1^ | 1073741824 (1GB) | Java Heap Size for the HDFS Secondary Name
 
 1. Applies to Cloudera distributions only.
 
+[[requirements-recommended-configuration-changes]]
 == Recommended Configuration Changes
 The following configuration changes are recommended but not required.
 
 NOTE: The Trafodion Installer does *not* make these changes.
 
+[[requirements-recommended-security-changes]]
+=== Recommended Security Changes
+
+The `trafodion` user ID should not be given other `sudo` privileges than what's specified in this manual. Also, we
+that this user ID is locked (`sudo passwd -l trafodion`) once the installation/upgrade activity has been completed.
```

Comment:
Missing verb? Did you mean, "Also, we checked that this user ID is locked..." Or perhaps you meant, "The Trafodion Installer checks that this user ID is locked..."